<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDoctor</class>
 <widget class="QWidget" name="AddDoctor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Doctor</string>
  </property>
  <widget class="QLabel" name="TitleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#4b0082;&quot;&gt;Add Doctor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumnberTxtBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string>+00 000 000 000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTxtBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>40</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QComboBox" name="specialtyComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Allergy and immunology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Adolescent medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Anaesthesiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pathology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cardiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paediatric cardiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cardiothoracic surgery</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Child and adolescent psychiatry and psychotherapy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Clinical neurophysiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dermatology-Venereology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Emergency medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Endocrinology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gastroenterology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>General practice</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Geriatrics</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Obstetrics and gynaecology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Health informatics</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hospice and palliative medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Infectious disease</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Internal medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Interventional radiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vascular medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Microbiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nephrology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neurology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neurosurgery</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nuclear medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Occupational medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ophthalmology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orthopaedics</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oral and maxillofacial surgery</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Otorhinolaryngology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paediatrics</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neonatology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paediatric surgery</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Physical medicine and rehabilitation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plastic, reconstructive and aesthetic surgery</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Podiatric medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pulmonology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Psychiatry</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Public Health</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Radiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sports medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neuroradiology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Radiotherapy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>General surgery</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Urology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vascular surgery</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="BirthDate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="gender">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAlphabetically</enum>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="usernameTxtBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameTxtBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="homeAddressTxtBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Home Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailAddressTxtBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>40</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Email Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cnpTxtBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>13</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>CNP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameTxtBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="genderLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>273</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Gender: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>71</y>
     <width>68</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;First Name: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>111</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Last Name: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>152</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Username: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="cnpLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>233</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;CNP: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="userPasswordLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>192</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Password: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="homeAddressLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>354</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Home Address: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateOfBirthLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>314</y>
     <width>82</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Date Of Birth: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="phoneNumnberLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>395</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Phone: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="specialtyLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>476</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Specialty: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailAddressLbl">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>435</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#4b0082;&quot;&gt;Email Address: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>firstNameTxtBox</tabstop>
  <tabstop>lastNameTxtBox</tabstop>
  <tabstop>usernameTxtBox</tabstop>
  <tabstop>passwordTxtBox</tabstop>
  <tabstop>cnpTxtBox</tabstop>
  <tabstop>BirthDate</tabstop>
  <tabstop>homeAddressTxtBox</tabstop>
  <tabstop>phoneNumnberTxtBox</tabstop>
  <tabstop>emailAddressTxtBox</tabstop>
  <tabstop>specialtyComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
